shared/src/business/useCases/generateSignedDocumentInteractor.test.js

```diff
@@ -28,7 +28,7 @@ describe('generateSignedDocument', () => {
     await PDFDocument.load(rotatedTestPdfDoc);
 
   beforeEach(() => {
-    jest.setTimeout(10000);
+    jest.setTimeout(30000);
```

Comment:
I worry about this being the right solution to fixing these tests. Nothing to change here, but 30 seconds -- where do we go from here?

Reply:
I believe we were having some issues with unexplained intermittent failures, and giving them more time (and bumping the container size) seemed to fix them.